config/sites/its.uiowa.edu/core.entity_view_display.node.service.default.yml

```diff
@@ -64,38 +75,248 @@ third_party_settings:
                 view_mode: view_mode
               formatter:
                 type: string
-                label: hidden
+                label: visually_hidden
                 settings:
                   link_to_entity: false
                 third_party_settings: {  }
             weight: -2
             additional: {  }
             third_party_settings: {  }
+          -
+            uuid: 4d222da8-3da7-4baf-a9a8-711b2293cf08
+            region: content
+            configuration:
+              id: 'field_block:node:service:field_service_audience'
+              label: null
+              label_display: null
+              provider: layout_builder
+              context_mapping:
+                entity: layout_builder.entity
+                view_mode: view_mode
+              formatter:
+                type: entity_reference_label
+                label: inline
+                settings:
+                  link: false
+                third_party_settings:
+                  field_delimiter:
+                    delimiter: ','
+            weight: -1
+            additional: {  }
+            third_party_settings: {  }
         third_party_settings:
           layout_builder_lock:
             lock: {  }
+          layout_builder_limit:
+            limit:
+              scope: disabled
+              scope_update: 'Update scope'
       -
         layout_id: layout_onecol
         layout_settings:
           label: ''
+          context_mapping: {  }
+          layout_builder_styles_style:
+            0: ''
+            1: section_margin_fixed_width_container
+            section_margin_remove_default_margins: section_margin_remove_default_margins
         components:
           -
-            uuid: 935aa0ea-f15f-4e36-871f-f06b5f7eaa4c
+            uuid: f15b0115-1699-4e58-823f-b90f656ed2d4
             region: content
+            configuration:
+              id: 'inline_block:uiowa_spacer_separator'
+              label: null
+              label_display: null
+              provider: layout_builder
+              view_mode: full
+              context_mapping: {  }
+              block_revision_id: 2243
+              block_serialized: null
+              type: uiowa_spacer_separator
+              uuid: ac628a5b-420d-4c66-bb9c-ecd8db8197ee
+            weight: 0
+            additional:
+              layout_builder_styles_style:
+                block_margin_default_removed: block_margin_default_removed
+            third_party_settings: {  }
```

Comment:
I think we are trying to move away from using the component. @bspeare your thoughts?